arrow-fx-coroutines/src/main/kotlin/arrow/fx/coroutines/ParTraverse.kt

```diff
@@ -35,8 +44,26 @@ suspend fun <A> Iterable<suspend () -> A>.parSequenceN(n: Long): List<A> =
  *
  * Cancelling this operation cancels all running tasks
  */
+@Deprecated("use parSequenceN with n as Int type", ReplaceWith("parTraverseN", "arrow.fx.coroutines.parSequenceN"))
 suspend fun <A> Iterable<suspend () -> A>.parSequenceN(ctx: CoroutineContext = EmptyCoroutineContext, n: Long): List<A> {
-  val s = Semaphore(n.toInt())
+  require(n <= Int.MAX_VALUE) { "n must be <= ${Int.MAX_VALUE}, was: $n" }
+  val s = Semaphore(Math.toIntExact(n))
```

Comment:
```suggestio
  val s = Semaphore(Math.toIntExact(n))
```

`Math.toIntExact` already checks for overflowing https://docs.oracle.com/javase/8/docs/api/java/lang/Math.html#toIntExact-long-.